.gitignore

```diff
@@ -45,4 +45,5 @@ osmt*.env
 /test/postman/osmt-auth.environment.json
 /test/postman/token.env
 /test/node_modules
+test/postman/node_modules/
```

Comment:
This change is unrelated. If it is needed, please push it as a separate commit.

Reply:
It is pushed as a separate commit to this feature branch

Reply:
If you remove your local test/postman/node_modules/ directory, it won't be rebuilt. Please remove this line from this PR.

Reply:
Updated